src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -968,6 +969,31 @@
 		return new FluxResume<>(source, fallback);
 	}
 
+
+	/**
+	 * Create a {@link Flux} that will fallback to a publish if the error exception matches the given type
+	 * failing to an error with the same that of original
+	 *
+	 * @param errorType exception type to match
+	 * @param source source sequence
+	 * @param fallback supplier called when the exceptions and errorType matches
+	 * @param <T> {@link Subscriber} type target
+	 * @return a {@link Flux} handling error if exceptions match otherwise an rejected Flux
+	 */
+	public final Flux<T> onErrorResumeWith(
+			Class<? extends Throwable> errorType,
+			Publisher<? extends T> source,
```

Comment:
The method doesn't use `this` so either it should be static or the `source` parameter should be removed and have `new FluxResume<>(this, ...)`.